### ISSUE TITLE
[Mirror] Adding an Australian Termux mirror

### ISSUE THREAD

**Issue by wale** [termux/termux-packages#15052]
I have recently introduced a new Termux mirror that is hosted in Sydney, Australia. The repositories that are mirrored are `termux-main`, `termux-main-21`, `termux-root` & `termux-x11`.

## Source list syntax
```
deb https://mirrors.wale.id.au/termux/termux-main stable main
deb https://mirrors.wale.id.au/termux/termux-root root stable
deb https://mirrors.wale.id.au/termux/termux-x11 x11 main
```

**Comment by wale** [comment]
It would be possible, but that would require more planned downtime on the mirror in case the rsync addition goes south, considering I also host mirrors for other distros. Do you mind if the rsync support is added later down the line?

**Comment by Grimler91** [comment]
> Do you mind if the rsync support is added later down the line?

Yeah that's fine, we can add it to our mirror list in any case (just not with the mirrorbits functionality), thanks!

**Comment by Grimler91** [comment]
Mirror is available as part of termux-tools 1.37.0, which will be available shortly!
